### ISSUE TITLE
Hdr Issue (screen dark when exiting)

### ISSUE THREAD

**Issue by JOHN30011887** [LostArtefacts/TRX#800]
I always have HDR enabled in windows 11 (not the auto hdr)

But with this game (Tomb1Main) latest version, the game is darker and when i exit the game it keeps my screen dark, where i then have to toggle Hdr off and on again, in windows

Is there any known fix for this issue?
i dont have this problem with any of the other tomb mains out there so im sort of puzzled 

Is there a way to force borderless, that would fix the issue?

**Comment by walkawayy** [comment]
Just tried this, and I get the same issue with a dark screen after exiting. Seems like it's a Windows issue from what I could google?

https://answers.microsoft.com/en-us/windows/forum/all/hdr-monitor-low-brightness-after-exiting-full/999f7ee9-7ba3-4f9c-b812-bbeb9ff8dcc1

https://old.reddit.com/r/Monitors/comments/tcdd11/very_dark_screen_after_playing_a_game_in_hdr_mode/

**Comment by JOHN30011887** [comment]
@walkawayy yeah it does look like its more a windows issue, i had to change Rpcs3 emulator and Yuzu to Borderless option, as fullscreen caused the darkness issue on them too while Borderless fixes it and keeps them bright/normal.
So this is the first and only Pc game iv had this issue with :(
